tools/cli/src/main/java/org/apache/carbondata/tool/DataFile.java

```diff
@@ -500,4 +503,7 @@ private double computePercentage(byte[] data, byte[] min, byte[] max, ColumnSche
     }
   }
 
+  public void close() throws IOException {
+    this.fileReader.finish();
```

Comment:
why not close?

Reply:
this is the `close` method for `fileReader`